package.json

```diff
@@ -86,11 +86,11 @@
     "npm-run-all": "4.1.5",
     "prettier": "2.2.1",
     "rimraf": "3.0.2",
-    "semantic-release": "17.4.2",
-    "serverless": "2.45.1",
+    "semantic-release": "19.0.2",
+    "serverless": "3.17.0",
     "serverless-dotenv-plugin": "3.1.0",
-    "serverless-offline": "6.8.0",
-    "serverless-s3-local": "0.6.7",
+    "serverless-offline": "8.7.0",
+    "serverless-s3-local": "0.6.22",
     "serverless-webpack": "5.3.5",
     "supertest": "6.0.1",
```

Comment:
Don't know whether it might be worth bumping some of the other packages like typescript and aws-sdk